src/documentation/pages/components/icon/examples/_semantic.vue

```diff
@@ -0,0 +1,61 @@
+<template lang="pug">
+<ComponentExample title="Semantic" titleSize="h4" id="semantic" padding="-p--0" :tabs="exampleTabs">
+  p.-text(slot="example-description")
+    | Use semantic colors to communicate meaning to users. Examples include displaying alerts, 
+    | form field validation, user status, application state and more. Use green (success) for positive,
+    | blue (info) for informative, red (danger) for negative, yellow (warning) for needs attention,
+    | and grey (muted) for neutral. Semantic colors should never be used for decorative purposes.
+  .chi-grid(slot="example")
```

Comment:
Can we use this?

```suggestion
  <Wrapper slot="example">
```

Within the Wrapper we may be able to use `.chi-grid` and other classes.